USING_VANAGON.md

```diff
@@ -0,0 +1,42 @@
+#Using Vanagon
+- Note that these docs are incomplete. Docs for all parts of vanagon are on their way
+
+##Creating a Vanagon Project for Windows
```

Comment:
Drop a space between your hashes and your headline. `## Headline` is easier on the eyes, and renders the same.